assets/javascripts/discourse/widgets/qa-comment.js

```diff
@@ -8,24 +8,86 @@ import { ajax } from "discourse/lib/ajax";
 import bootbox from "bootbox";
 import I18n from "I18n";
 
+createWidget("qa-comment-editor", {
```

Comment:
Nit: one widget per file would be nice

Mostly for when I look at logic I'd also refer to the file name to check if it belongs in that "class".

Reply:
Good point :+1: I've split the widgets into one per file.